src/lib/plugins/login.js

```diff
@@ -175,13 +200,10 @@ module.exports.player = async function (player, serv, settings) {
         ping: otherPlayer._client.latency
       }))
     })
-    setInterval(() => player._client.write('player_info', {
```

Comment:
looks like this is completely removed even for before 1.19 ; is this expected?

Reply:
This sends the latency for all the clients but the previous player_info packet already sends the latencies in its payload with action mode 0

Back to 1.8 https://github.com/PrismarineJS/minecraft-data/blob/master/data/pc/1.8/proto.yml#L565

I think it may have been added because it's what vanilla did at some point but it wasn't doing that for 1.17. I could re-add it behind a flag for <1.17

Reply:
Done